pkg/osquery/runtime/runner.go

```diff
@@ -419,7 +419,7 @@ func (r *Runner) launchOsqueryInstance() error {
 				// better for a Limiter
 				maxHealthChecks := 5
 				for i := 1; i <= maxHealthChecks; i++ {
-					if err := o.Healthy(); err != nil {
+					if err := r.Healthy(); err != nil {
```

Comment:
I think this raises an question -- Why do we have both `o.Healthy` and `r.Healthy` and should one be a wrapper over the other?